### PR TITLE
Add Software AG

### PR DESCRIPTION
**Pull request by lbadenhop**
Thank you for your contribution to the Amplify-Central repo.

## Describe the changes

Enter a brief description of your changes to communicate to the reviewers what you changed and why.

## Deploy preview link

Please add the deploy preview link to the **specific page** that you've changed.

## Checklist for contributors

Before submitting this PR:

* Verify that all status checks have passed
* For more information on the Markdown linter rules, see [DavidAnson/markdownlint](https://github.com/DavidAnson/markdownlint)
* For first-time contributors, ensure that you have signed the [Axway CLA](https://cla.axway.com/)

### CODE REVIEW

**Comment by docsbuilder** [comment]
### <span aria-hidden="true">✅</span> Deploy preview is ready!

 |  Name | Link |
 |---------------------------------|------------------------|
 |<span aria-hidden="true">🔨</span> Latest commit | 267a0af |
 |<span aria-hidden="true">😎</span> Deploy preview | http://PR-521.amplify-central.opendocs-builder.pcloud.axway.int |

File: content/en/docs/connect_manage_environ/_index.md [hunk 1]
```diff
@@ -56,10 +56,10 @@ Using agents is the recommended way to add API services to your environment. Whe
...
+| **Quota Enforcement**          | No                                         | Quotas are attached to a Usage plan                         | Rate limiting plugin added to Consumer                     | No                       | No             |
+| **Traceability**               | No                                         | Yes                         | Using the Kong HTTP log plugin                             | No                       | No             |
+| **Platform Usage**             | Yes                                        | Yes                         | Yes                                                        | No                       | Yes            |
+| **Transaction Metrics**        | No                                         | Yes                         | Yes                                                        | No                       | Yes            |
 | **Transaction Event Sampling** | No                                         | No                         | Yes                                                        | No                       | No             |
```
Comment (jcollins-axway) [inline content/en/docs/connect_manage_environ/_index.md:63]: Event Sampling -> Yes

Comment (robertengaxway) [reply content/en/docs/connect_manage_environ/_index.md]: edits made

File: content/en/docs/connect_manage_environ/_index.md [hunk 2]
```diff
@@ -56,10 +56,10 @@ Using agents is the recommended way to add API services to your environment. Whe
...
-| **Quota Enforcement**          | No                                         | No                         | Rate limiting plugin added to Consumer                     | No                       | No             |
-| **Traceability**               | No                                         | No                         | Using the Kong HTTP log plugin                             | No                       | No             |
-| **Platform Usage**             | Yes                                        | No                         | Yes                                                        | No                       | Yes            |
-| **Transaction Metrics**        | No                                         | No                         | Yes                                                        | No                       | Yes            |
+| **Quota Enforcement**          | No                                         | Quotas are attached to a Usage plan                         | Rate limiting plugin added to Consumer                     | No                       | No             |
```
[the remark this answers is not in the document]
Comment (robertengaxway) [reply content/en/docs/connect_manage_environ/_index.md]: edits made

**Review by robertengaxway — APPROVED** [review]

**Pull request merged**